case.jsonld

```diff
@@ -1705,6 +1705,118 @@
+                    "uco-observable:application": {
+                        "@id": "kb:63455f6a-3e0a-58d2-b425-68030c41a62a"
+                    }
+                }
+            ]
+        },
+        {
+            "@id": "kb:b6326bd9-8691-59c5-b126-9ca2be995d78",
+            "@type": "uco-observable:ObservableObject",
+            "uco-core:hasFacet": [
+                {
+                    "@id": "kb:82bd053b-bb47-5931-a59e-bcb6b451d048",
+                    "@type": "uco-observable:ApplicationFacet",
+                    "uco-observable:applicationIdentifier": "iPhoneNetworkDataUsage"
+                }
+            ]
+        },
+        {
+            "@id": "kb:1b700918-7f2d-55ad-acd8-fba19ce9e7aa",
+            "@type": "uco-observable:ObservableAction",
+            "uco-action:actionCount": {
+                "@type": "xsd:nonNegativeInteger",
+                "@value": "3965"
+            },
+            "uco-action:actionStatus": {
+                "@type": "uco-vocabulary:ActionStatusTypeVocab",
+                "@value": "Complete/Finish"
+            },
+            "uco-core:description": "Network log entry extraction of all lines",
+            "uco-core:name": "Network log entry",
+            "uco-action:endTime": {
+                "@type": "xsd:dateTime",
+                "@value": "2024-02-29T12:43:44+00:00"
+            },
+            "uco-action:startTime": {
+                "@type": "xsd:dateTime",
+                "@value": "2024-02-29T12:28:49+00:00"
+            },
+            "uco-action:instrument": {
+                "@id": "kb:dae7ab48-4877-5aeb-83be-51c67fa53493"
+            },
+            "uco-action:location": {
+                "@id": "kb:f0e14e08-35aa-5381-b2f2-5822f93163b7"
+            },
+            "uco-action:object": {
+                "@id": "kb:1506e4eb-8c84-5ffc-9329-3cf998ca145b"
+            },
+            "uco-action:participant": {
+                "@id": "kb:0db6f576-b92c-520d-9e1a-445eb103254b"
+            },
+            "uco-action:performer": {
+                "@id": "kb:0db6f576-b92c-520d-9e1a-445eb103254b"
+            },
+            "uco-action:result": {
+                "@id": "kb:1dec274d-705a-5c0e-8b03-226f2e227501"
+            }
+        },
+        {
+            "@id": "kb:28779a1c-3868-5089-9408-e989937cf375",
+            "@type": "uco-observable:ObservableObject",
+            "uco-core:hasFacet": [
+                {
+                    "@id": "kb:075ee909-b08c-5f4a-9c43-08a00829d72d",
+                    "@type": "uco-observable:EventRecordFacet",
+                    "uco-observable:eventRecordID": "geod/AlexisBarreyat.BeReal",
+                    "uco-observable:eventRecordRaw": "[CDATA[Wifi In:0, Wifi Out:0, Wan In:37847, 10689999, Wan Out:18956]]",
+                    "uco-observable:eventRecordServiceName": "BeReal",
+                    "uco-observable:eventRecordText": "Wifi In:0, Wifi Out:0, Wan In:37847, 10689999, Wan Out:18956",
```

Comment:
I suspect what you have in `eventRecordText` should actually be what goes into `eventRecordRaw`.  The "CDATA" tells me you pulled this from an XML file somewhere.  "CDATA" is an XML-specific syntactic component to preserve text streams, and should not be ported into other syntaxes like Turtle or JSON-LD.

I think what should go into `eventRecordText` is supposed to be a text render supplied elsewhere in the source data, but I admit I'm a little fuzzy on this property.